.github/workflows/scheduled-build.yml

```diff
@@ -13,4 +13,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Curl request
-        run: curl -X POST -d {} ${{ secrets.NETLIFY_BUILD_HOOK }}
+        env:
+          NETLIFY_BUILD_HOOK: ${{ secrets.NETLIFY_BUILD_HOOK }}
+        run: curl -X POST -d {} NETLIFY_BUILD_HOOK
```

Comment:
This has nothing to do with making the logo a link. Something got messed up with this PR